### ISSUE TITLE
[Check] Mass assignment in Django

### ISSUE THREAD

**Issue by minusworld** [returntocorp/sgrep-rules#289]
**Check Description**
This pattern ([src](https://gist.github.com/jsocol/3217262)):
```python
from django.shortcuts import render
from myapp.models import Whatzit

def create_whatzit(request):
    Whatzit.objects.create(**request.POST)
    return render(request, 'created.html')

def update_whatzit(request, id):
    whatzit = Whatzit.objects.filter(pk=id)
    whatzit.update(**request.POST)
    whatzit.save()
    return render(request, 'saved.html')
```

is bad because it allows one to update fields that were originally not intended to be updated. An example of this is a boolean field on a User model called `isAdmin` which could be injected due to the "mass assignment" of parameters when creating the model.  For more info, see https://cheatsheetseries.owasp.org/cheatsheets/Mass_Assignment_Cheat_Sheet.html.

- [x] This ticket has links, references, or examples.
- [x] Your check has true positive and true negative test cases.
- [x] Your check has been labeled with true positive and false positive findings on at least 10 repositories, and the link is pasted in the ticket. If there are no findings, paste the triager link anyway.
- [ ] Your check PR has been reviewed and merged.
Now, close the ticket!

**Comment by minusworld** [comment]
https://dev.massive.ret2.co/jobs/828?page=1&sort_on=repo_url&sort_by=asc --> no findings
